omod/src/main/resources/config.xml

```diff
@@ -89,6 +89,13 @@
 			Format used by Javascript functions for dates that have a time component
 		</description>
 	</globalProperty>
+
+	<globalProperty>
+		<property>uiframework.timezone.conversions</property>
```

Comment:
Let's name it as if it was a Core GP already: `timezone.conversions` and do the same in HFE so that all modules actually rely on the same Core-to-be GP name.

Maybe also add an XML comment to say that this should be removed once this GP is provided by Core itself and that UIFR depends on the version of Core that has it.